packaging/datadog-agent/source/supervisord.conf

```diff
@@ -40,5 +40,12 @@ redirect_stderr=true
 priority=998
 startsecs=3
 
+[program:jmxfetch]
+command=python jmxfetch.py
+stdout_logfile=jmxfetch.log
```

Comment:
This is not the right path for the log. It should be supervisord/logs/jmxfetch.log